src/PerfView/EventViewer/EventWindow.xaml.cs

```diff
@@ -605,7 +561,149 @@ private void DoShowEventCounterGraph(object sender, ExecutedRoutedEventArgs e)
             string uri = "file:///" + html.Replace('\\', '/').Replace(" ", "%20");
             Process.Start(uri);
         }
-            
+
+        private const string PayloadToken = "Payload=\"{";
```

Comment:
For .NET Core, the Payload format is different, PayloadToken should be "Payload:{"

Example:

HasStack="True" ThreadID="3,120" ProcessorNumber="0" ="{ Payload:{ Name:"cpu-usage", DisplayName:"CPU Usage", Mean:0, StandardDeviation:0, Count:1, Min:0, Max:0, IntervalSec:0.0010742, Series:"Interval=1000", CounterType:"Mean", Metadata:"", DisplayUnits:"" } }"

Reply:
I don't know from where you got this payload @felipepessoto but it is not what is parsed by perfview to generate the graphs. 
Click one of the */EventCounters event and look at its content with ALT+D/right click | Dump Event and you'll see that Payload= is present also for .NET Core 3.0.
You could also set a breakpoint at the change position and look at the string being parsed.

Reply:
@chrisnas, I got it from dotnet-trace:

dotnet-trace collect --process-id <PID> --providers System.Runtime:ffff:5:EventCounterIntervalSec=1,Microsoft-Windows-DotNETRuntime:0:1,Microsoft-DotNETCore-SampleProfiler:0:1